# Overview
------

### Multilayer
------
![](images/multilayer/logo.png)

Coming soon.

#### Tetramer
------
<center><a href="http://www.ngs-qc.org/tetramer/index.php"><img src="images/tetramer/logo.png" width="25%" height="25%"></a></center>
------
<a href="https://www.nature.com/articles/s41540-018-0066-z"><img src="images/tetramer/doi_tetramer.png" width="25%" height="25%"></a>

<a href="http://apps.cytoscape.org/apps/tetramer"><img src="images/tetramer/cytoscape_tetramer.png" width="25%" height="25%"></a>


<p align="justify">
<strong>TETRAMER</strong> provides a user-friendly framework for the <strong>reconstruction of cell fate transition-specific gene regulatory networks (GRNs)</strong> by integrating user-provided temporal transcriptomes with generic GRNs derived from (i) the analysis of multiple publicly available gene expression profiles associated to several mouse or human cell type/tissues (CellNet; http://cellnet.hms.harvard.edu/); (ii) the genome-wide mapping of human promoters and enhancers in multiple cell type/tissues from CAGE data generated by the FANTOM5 consortium (regulatory circuits http://regulatorycircuits.org/); as well as (iii) the systematic analysis of most publicly available ChIP-sequencing data corresponding to TF-binding in a variety of human or mouse cell type/tissues: http://ngs-qc.org/. Furthermore, <strong>TETRAMER provides an iterative approach for interrogating the capacity of each TF</strong>, retrieved on the GRN, <strong>to drive cell fate transformation.</strong> For it the temporal transcriptional regulation cascade derived from each TF is scrutinized as a way to verify its influence on the reconstitution of the differential gene expression patterns associated to the cell fate transformation.
</p>
